### PR TITLE
Custom word

### PR DESCRIPTION
**Pull request by RogerDMK**
# Description
Added the custom word, if the relevant game mode is selected, allowing the user to specify which word is to be guessed. Added the creation of the game in the database.
Related to #59

Shows the number of players selected in the game creation form

### CODE REVIEW

File: src/public/lobby.js [hunk 1]
```diff
@@ -38,19 +57,30 @@ socket.on('get_game_id', (gameID) => {
...
-        //console.log(`Custom game with ${numPlayers} players.`)
-        gameType = 'custom'
+    const numPlayers = $("#numPlayers").val()
+    const gameType = $("#gameMode").val()
+    let modeChosen = 1
+    
+    // Checks if the custom word chosen is valid if the game mode is custom
+    if ($("#gameMode").val() === "custom") {
+        modeChosen = 2
+        if ($("#customWord").val() != "") {
```
Comment (UntidyRAM) [inline src/public/lobby.js:67]: Should this not be ```!==```?

File: src/public/lobby.js [hunk 2]
```diff
@@ -38,19 +57,30 @@ socket.on('get_game_id', (gameID) => {
...
-        gameType = 'custom'
+    const numPlayers = $("#numPlayers").val()
+    const gameType = $("#gameMode").val()
+    let modeChosen = 1
+    
+    // Checks if the custom word chosen is valid if the game mode is custom
+    if ($("#gameMode").val() === "custom") {
+        modeChosen = 2
+        if ($("#customWord").val() != "") { 
+            if ($("#customWord").val().length != 5) {
```
Comment (UntidyRAM) [inline src/public/lobby.js:68]: Should this also not be ```!==```?

File: src/public/lobby.js [hunk 3]
```diff
@@ -38,19 +57,30 @@ socket.on('get_game_id', (gameID) => {
...
+                window.alert("Oi, words have to be 5 letters, so I think you should do that.")
+                $("#customWord").focus()
+                return
+            }
+        } else {
+            window.alert("If you didn't want to pick a word, why did you pick custom... Standard is fun cause it's a random word.")
```
Comment (UntidyRAM) [inline src/public/lobby.js:74]: I love this! 😆 
Maybe we can replace the .alert with a bootstrap component so it looks better?

**Review by UntidyRAM — APPROVED** [review]
Looks great! Nice job.
Please just give the comments I made a read and make any changes if anything I said is applicable.

File: src/public/lobby.js (hunk 3 above)
Comment (RogerDMK) [reply src/public/lobby.js:74]: We definitely can, I think it does currently fit with the current style of socket popups tho. I think we can definitely change this when doing general styling though.

File: src/public/lobby.js (hunk 1 above)
Comment (RogerDMK) [reply src/public/lobby.js:67]: I will change that now, shouldn't change functionality but it is always good to make sure they are the same type!

File: src/public/lobby.js (hunk 2 above)
Comment (RogerDMK) [reply src/public/lobby.js:68]: Same as the previous, but will change it now.

**Pull request merged**